frontend/lib/pages/data-driven-onboarding.tsx

```diff
@@ -317,21 +320,24 @@ const ACTION_CARDS: ActionCardPropsCreator[] = [
     };
   },
   function evictionFreeNyc(data): ActionCardProps {
+    // Default content temporarily implemented during COVID-19 Outbreak
+    const covidMessage = <>
+      An Eviction Moratorium is in place in NY State due to the Covid-19 public health crisis. 
+      All courts that hear eviction cases are closed. This means you <b>cannot be evicted for any reason</b>. 
+    </>
+    const covidCtaText = "Learn more";
```

Comment:
Just FYI, "Learn more" is one of those not-great-for-accessibility link texts, because assistive tech (AT) users who are skimming a page by going through each link's text are just going to hear "learn more" without any context...  It's not a dealbreaker though and there are ways for us to append screenreader-only text to it so that for sighted users it says "Learn more" but for AT users it will say "Learn more about the eviction moratorium"... but because of the abstraction layer we've put in here, that won't be super straightforward.

So in other words, if it's easy to change "Learn more" to something a bit more accessible that also wouldn't be weird to sighted users, that'd be ideal, but otherwise I wouldn't worry about it right now (though it would be good to file an issue so we can address it later... same w/ the "Learn more" on the banner from #1047 for that matter, I overlooked it when I reviewed that one!).

Reply:
Marking this as an issue to overhaul later! https://github.com/JustFixNYC/tenants2/issues/1067